pkg/k8s/scheme.go

```diff
@@ -0,0 +1,23 @@
+package k8s
+
+import (
+	nodenetworkconfig_v1alpha "github.com/Azure/azure-container-networking/crd/nodenetworkconfig/api/v1alpha"
+	overlayextensionconfig_v1alpha1 "github.com/Azure/azure-container-networking/crd/overlayextensionconfig/api/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// NewScheme builds and returns k8s schemes used by ALB Controller.
+func NewScheme() (*runtime.Scheme, error) {
+	s := runtime.NewScheme()
+	sb := runtime.SchemeBuilder{
+		// Azure CNI CRDs
+		overlayextensionconfig_v1alpha1.AddToScheme,
```

Comment:
are these two crds present on the cluster by default ?

Reply:
These two are suppose to be present only on overlay clusters.